dipy/align/sumsqdiff.pyx

```diff
@@ -34,20 +34,40 @@ cdef void _solve_2d_symmetric_positive_definite(double[:] A, double[:] y,
     out[0] = (y[0] - A[1] * out[1]) / A[0]
 
 
-def solve_2d_symmetric_positive_definite(double[:] A, double[:] y,
-                                               double det,
-                                               double[:] out):
+def solve_2d_symmetric_positive_definite(A, y, double det):
     r"""
-    Wrapper for _solve_2d_symmetric_positive_definite (see documentation above)
+    Wrapper for _solve_2d_symmetric_positive_definite
```

Comment:
Since this is the public interface, it's confusing to call it a "wrapper". Can we change that to say what the 'wrapped' function does?